Keas.Mvc/Controllers/TeamAdminController.cs

```diff
@@ -206,122 +206,6 @@ public async Task<IActionResult> RemoveRoles(string userId, int[] roles)
-        public async Task<IActionResult> CreateMember(CreateMemberPersonModel person)
-        {
-            if (ModelState.IsValid)
-            {
-                var exisitngPerson =
-                    await _context.People.SingleOrDefaultAsync(p => p.Team.Slug == Team && p.UserId == person.User.Id);
-                if (exisitngPerson != null)
-                {
-                    Message = "User is already a member of this team!";
-                    return RedirectToAction(nameof(EditMember), new {id = exisitngPerson.Id});
-                }
-                var user = person.User;
-                var existingUser = await _context.Users.Where(x => x.Id == user.Id).AnyAsync();
-                if (!existingUser)
-                {
-                   var createdUser = await _userService.CreateUserFromEmail(person.User.Email);
-                }
-                var team = await _context.Teams.SingleAsync(t => t.Slug == Team);
-                var newPerson = new Person
-                {
-                    Team = team,
-                    FirstName = user.FirstName,
-                    LastName = user.LastName,
-                    Email = user.Email,
-                    UserId = user.Id,
-                    Group = person.Group,
-                    Title = person.Title,
-                    HomePhone = person.HomePhone,
-                    TeamPhone = person.TeamPhone
-                };
-                _context.People.Add(newPerson);
-                await _context.SaveChangesAsync();
-                Message = newPerson.User.Name + " added to team.";
-
-
-                return RedirectToAction(nameof(Members));
-            }
-            return View();
-        }
-
-        public async Task<IActionResult> EditMember(int id)
-        {
-            var model = await _context.People.Include(p => p.User).SingleAsync(x => x.Id == id);
-            return View(model);
-        }
-
-        [HttpPost, ActionName("EditMember")]
-        public async Task<IActionResult> EditMemberPost(int id)
-        {
-            if (id == null)
-            {
-                return NotFound();
-            }
-            var personToEdit = await _context.People.SingleAsync(x => x.Id == id);
-
-           
-            if (await TryUpdateModelAsync<Person>(personToEdit, "", t => t.FirstName, t => t.LastName, t => t.Active, t=> t.Group, t=> t.Title, t=> t.HomePhone, t=> t.TeamPhone))
-            {
-                try
-                {
-                    await _context.SaveChangesAsync();
-                    return RedirectToAction(nameof(Members));
-                }
-                catch
-                {
-                    ModelState.AddModelError("", "Unable to save changes.");
-                }
-            }
-            Message = "Something failed in the update.";
-             var model = await _context.People.Include(p => p.User).SingleAsync(x => x.Id == id);
-            return View(model);
-            
-        }
-
```

Comment:
@jcubbage If you are removing these methods, did you want to remove the views as well?